.github/ISSUE_TEMPLATE.md

```diff
@@ -5,8 +5,8 @@
 
 ##### Reproduction Steps
 1. [Step 1]
-1. [Step 2]
-1. [Step 3]
+2. [Step 2]
```

Comment:
This may be true, although I feel people may not know this. I know I change it every single time I post an issue. Can we set it to 1.2.3. so that people don't question it? Doesn't seem like it negatively impacts anything.